notebooks/atlas-and-kai/meta.toml

```diff
@@ -1,4 +1,5 @@
 [meta]
+authors=["singlestore"]
```

Comment:
I'm concerned that doing more of this stuff will make it too tricky to submit a new notebook.

I think we should just have `author_name`, `author_title`, etc. in each `meta.toml` file. It is duplicated yes but we can validate with the checks and most people only add one notebook at a time so not a big deal.

Reply:
Ah I see, ok yeah let's leave it like this then